x/oracle/module.go

```diff
@@ -104,22 +124,32 @@ func (AppModule) QuerierRoute() string {
 }
 
 // NewQuerierHandler returns the oracle module sdk.Querier.
-func (am AppModule) NewQuerierHandler() sdk.Querier {
+func (am AppModule) LegacyQuerierHandler(legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
 	return nil
 }
 
+// RegisterServices registers module services.
+func (am AppModule) RegisterServices(cfg module.Configurator) {
+	// types.RegisterMsgServer(cfg.MsgServer(), keeper.NewMsgServerImpl(am.keeper))
+	// querier := keeper.Querier{Keeper: am.keeper}
+	// types.RegisterQueryServer(cfg.QueryServer(), querier)
+}
+
 // InitGenesis performs genesis initialization for the oracle module. It returns
 // no validator updates.
-func (am AppModule) InitGenesis(ctx sdk.Context, data json.RawMessage) []abci.ValidatorUpdate {
-	var genesisState GenesisState
-	ModuleCdc.MustUnmarshalJSON(data, &genesisState)
+func (am AppModule) InitGenesis(ctx sdk.Context, cdc codec.JSONMarshaler, data json.RawMessage) []abci.ValidatorUpdate {
+	var genesisState types.GenesisState
+
+	cdc.MustUnmarshalJSON(data, &genesisState)
+
 	return InitGenesis(ctx, am.keeper, genesisState)
 }
 
 // ExportGenesis returns the exported genesis state as raw bytes for the oracle
 // module.
-func (am AppModule) ExportGenesis(ctx sdk.Context) json.RawMessage {
```

Comment:
@juniuszhou  Is there a reason we were returning for export ? If so we might want to keep it that way ?

Reply:
Actually , I see the the Init genesis is setting  the values , so I guess exporting it makes senses